docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -348,6 +348,310 @@ and a file system location of ``/var/my_tables/test_table``::
+  * - ``parent_id``
+    - ``bigint``
+    - The identifier for the parent snapshot
+  * - ``is_current_ancestor``
+    - ``boolean``
+    - Whether or not this snapshot is an ancestor of the current snapshot
+
+
+``$snapshots`` table
+^^^^^^^^^^^^^^^^^^^^
+
+The ``$snapshots`` table provides a detailed view of snapshots of the
+Iceberg table.
+
+You can retrieve the information about the snapshots of the Iceberg table
+``test_table`` by using the following query::
+
+    SELECT * FROM "test_table$snapshots"
+
+.. code-block:: text
+
+     commited_at        | snapshot_id          | parent_id            | operation          |  manifest_list                                                                                                                           |   summary
+    --------------------+----------------------+----------------------+--------------------+------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    2022-01-10 08:11:20 | 8667764846443717831  |  <null>              |  append            |   hdfs://hadoop-master:9000/user/hive/warehouse/test_table/metadata/snap-8667764846443717831-1-100cf97e-6d56-446e-8961-afdaded63bc4.avro | {changed-partition-count=0, total-equality-deletes=0, total-position-deletes=0, total-delete-files=0, total-files-size=0, total-records=0, total-data-files=0}
+    2022-01-10 08:11:34 | 7860805980949777961  | 8667764846443717831  |  append            |   hdfs://hadoop-master:9000/user/hive/warehouse/test_table/metadata/snap-7860805980949777961-1-faa19903-1455-4bb8-855a-61a1bbafbaa7.avro | {changed-partition-count=1, added-data-files=1, total-equality-deletes=0, added-records=1, total-position-deletes=0, added-files-size=442, total-delete-files=0, total-files-size=442, total-records=1, total-data-files=1}
+
+
+The output of the query has the following columns:
+
+.. list-table:: Snapshots columns
+  :widths: 20, 30, 50
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``commited_at``
+    - ``timestamp(3) with time zone``
+    - The time when the snapshot became active
+  * - ``snapshot_id``
+    - ``bigint``
+    - The identifier for the snapshot
+  * - ``parent_id``
+    - ``bigint``
+    - The identifier for the parent snapshot
+  * - ``operation``
+    - ``varchar``
+    - The type of operation performed on the Iceberg table.
+      Supported operation types in Iceberg are
+
+      * ``append`` when new data is appended,
+      * ``replace`` when files are removed and replaced without changing the data in the table,
+      * ``overwrite`` when new data is added to overwrite existing data,
+      * ``delete`` when data is deleted from the table  and no new data is added
```

Comment:
Nit: As this looks like an unordered sublist, are the commas at the end of each statement standard?